File: claudecodeide/add_file.py
import sys
from pbxproj import XcodeProject

project_path = 'claudecodeide.xcodeproj/project.pbxproj'
project = XcodeProject.load(project_path)

file_path = 'claudecodeide/Sources/Utilities/AgentReasoningParser.swift'

# Find the target
target = project.get_target_by_name('claudecodeide')
if not target:
    print("Target not found")
    sys.exit(1)

# Add the file to the project and target
project.add_file(file_path, force=False, target_name='claudecodeide')

project.save()
print("Added file successfully")

File: claudecodeide/add_file.rb
require 'xcodeproj'
project_path = 'claudecodeide.xcodeproj'
project = Xcodeproj::Project.open(project_path)
target = project.targets.first

group = project.main_group.find_subpath(File.join('claudecodeide', 'Sources', 'Utilities'), true)
group.set_source_tree('<group>')

file_path = File.join('claudecodeide', 'Sources', 'Utilities', 'AgentReasoningParser.swift')
file_ref = group.new_reference(file_path)

target.add_file_references([file_ref])

project.save
